src/utils/constants/urls.ts

```diff
@@ -28,6 +28,8 @@ export const PrefixedUrls = Object.fromEntries(
     Object.entries(Urls).map(([key, value]) => [key, servicePathPrefix + value])
 ) as Readonly<{ [K in keyof typeof Urls]: string }>;
 
+export const generatePathWithCompany = (path: string) => `${path}/company/:companyNumber`;
```

Comment:
The company number should come before accounts filing to match other services
`domain/company/12345678/accounts-filing`